src/readinp.F90

```diff
@@ -51,9 +51,6 @@ subroutine readinp
  read (12,*) rand_start
  read (12,*)
  read (12,*)
- read (12,*) save_probe_steps, save_span_average_steps
- read (12,*)
- read (12,*)
```

Comment:
no longer need to read IO information from config files for these parameters since they are unused